playbooks/openshift-glusterfs/private/gluster_hosts.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Configure GlusterFS hosts
+  hosts: "{{ l_glusterfs_hosts | default('glusterfs') }}"
+  tasks:
+  - import_role:
+      name: openshift_storage_glusterfs
+      tasks_from: firewall.yml
+    when: openshift_storage_glusterfs_is_native | default(True) | bool
+  - import_role:
+      name: openshift_storage_glusterfs
+      tasks_from: kernel_modules.yml
+  - import_role:
+      name: openshift_storage_glusterfs
+      tasks_from: host_services.yml
+    when: openshift_storage_glusterfs_is_native | default(True) | bool
```

Comment:
These three imports are the same as the ones for glusterfs_registry nodes. Why not combine the two lists into one and only specify this play once? If there's new nodes in one and not the other, only that one set will be run. If there's new nodes in both, both sets will be covered. What am I missing?

Reply:
This is copy pasta from what was being done previously, just refactored a little bit to limit host group to intersection of new_nodes during scaleup.

I suppose I can refactor it.